target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -235,6 +236,7 @@ ramips_setup_macs()
 	case $board in
 	aigale,ai-br100|\
 	asus,rt-ac51u|\
+	asus,rt-ac54u|\
```

Comment:
Please check whether the WAN MAC address you find in vendor firmware is also found in flash (check the locations given in my comment above).

What's the MAC address assignment on vendor firmware (LAN, WAN, 2g, 5g)?